Sources/Commands/SwiftPackageTool.swift

```diff
@@ -466,9 +466,20 @@ extension SwiftPackageTool {
         @Option(help: "text | dot | json | flatlist")
         var format: ShowDependenciesMode = .text
 
+        @Option(name: [.long, .customShort("o") ], 
+                help: "The absolute or relative path to output the resolved dependency graph.")
+        var outputPath: AbsolutePath?
+
         func run(_ swiftTool: SwiftTool) throws {
             let graph = try swiftTool.loadPackageGraph()
-            dumpDependenciesOf(rootPackage: graph.rootPackages[0], mode: format)
+            let stream : OutputByteStream
```

Comment:
let stream = outputPath.map { try LocalFileOutputByteStream($0) } ?? TSCBasic.stdoutStream.stream

Reply:
Updated, two _try_ were required, once the exception was being thrown by the map